File: app/src/main/java/com/meiji/daily/interfaces/IOnItemClickListener.java
package com.meiji.daily.interfaces;

import android.view.View;

/**
 * Created by Meiji on 2016/11/18.
 */

public interface IOnItemClickListener {

    /**
     * RecyclerView Item点击事件
     *
     * @param view
     * @param position
     */
    void onClick(View view, int position);
}

File: app/src/main/java/com/meiji/daily/listener/IOnItemClickListener.kt
package com.meiji.daily.listener

import android.view.View

/**
 * Created by Meiji on 2016/11/18.
 */

interface IOnItemClickListener {

    /**
     * RecyclerView Item点击事件
     *
     * @param view
     * @param position
     */
    fun onClick(view: View, position: Int)
}
